source/locale/pt_BR/LC_MESSAGES/internal-communication.po

```diff
@@ -649,20 +649,110 @@ msgid "Kafka-WS"
 msgstr ""
 
 #: ../../source/internal-communication.rst:558
+#, fuzzy
 msgid ""
-"*Kafka WebSocket* service allows the users to retrieve conditional and/or"
-" partial real time data from a given dojot topic in its internal Kafka "
-"cluster."
+"*Kafka WebSocket* allows the users to retrieve data from a given dojot "
+"topic in a Kafka cluster, this retrieval can be conditional and/or "
+"partial. It works with pure websocket connections, so you can create "
+"websocket clients in any language you want as long as they support RFC "
+"6455."
 msgstr ""
-"O serviço *Kafka WebSocket* permite aos usuários a recuperação parcial e/ou "
-"condicional de dados em tempo real de um determinado tópico do cluster Kafka "
-"interno da dojot."
+"O serviço *Kafka WebSocket* permite aos usuários a recuperação de dados "
+"de um topico da dojot em um Cluster Kafka, essa recuperação pode ser "
+"condicional e/ou parcial. Funciona com uma conexão websocket pura, para "
+"que clientes websocket possam ser criados em qualquer linguagem que tenha"
+" suporte para o RFC 6455."
 
-#: ../../source/internal-communication.rst:562
+#: ../../source/internal-communication.rst:564
+msgid "Connecting to the service"
+msgstr "Conectando com o serviço"
+
+#: ../../source/internal-communication.rst:566
+#, fuzzy
+msgid ""
+"The connection is done in two steps, you must first obtain a single-use "
+"ticket through a REST request, then, be authorized to connect to the "
+"service through a websocket."
+msgstr ""
+"A conexão é realizada em dois passos, primeiro obtendo um ticket de uso "
+"único mediante uma requisição REST, depois ser autorizado para conectar "
+"com o serviço via websocket"
+
+#: ../../source/internal-communication.rst:571
+msgid "First step: Get the single-use ticket"
+msgstr "Primeiro passo: obter um ticket de uso único"
+
+#: ../../source/internal-communication.rst:572
+msgid ""
+"A ticket allows the user to subscribe to a dojot topic. To obtain it is "
+"necessary to have a JWT access token that is issued by the platform's "
+"Authentication/Authorization service. Ticket request must be made by REST"
+" at the endpoint <base-url>/kafka-ws/v1/ticket using the HTTP GET verb. "
+"The request must contain the header Authorization and the JWT token as "
+"value, according to the syntax:"
+msgstr ""
+"Um ticket permite que o usúario se subscrever com um topico da dojot. "
+"Para obter o ticket é necessário ter um token JWT gerado pelo serviço de "
+"Autenticação/Autorização da plataforma. a requisição REST do ticket deve "
+"ser redirecionada para o endpoint <base-url>/kafka-ws/v1/ticket usando o "
+"verbo HTTP GET. A requisição deve ter o cabeçalho Authorizatione o token "
```

Comment:
> verbo HTTP GET. A requisição deve ter o cabeçalho Authorization **e** o token

the space between words is missing.
That's why we ask to have space at the end (or at the beginning) of the sentences before closing the quotes.